src/main/java/com/amihaiemil/zold/CharSequences.java

```diff
@@ -0,0 +1,37 @@
+package com.amihaiemil.zold;
+
+import java.util.Objects;
+
+/**
+ * Provides character sequences operations.
+ * @author Ammar Atef (ammar.atef45@gmail.com)
+ * @version $Id$
+ * @since 0.0.3
+ */
+public final class CharSequences {
```

Comment:
@ammaratef45 , thanks for the changes! One minor: this can be package private I believe (we do not want to expose this for the end users, don't we?)

Reply:
@denis-konovalyenko 👍 indeed